extras/docker/generate.py

```diff
@@ -54,14 +64,14 @@ def render_template(template_filename, context):
         "redhat-ubi-9.4": "microdnf upgrade -y && microdnf install -y openssl",
     }
     post_install_commands = {
-        "alpine3.19": "",
-        "bookworm": "\nRUN ldconfig\n",
-        "noble": "\nRUN ldconfig\n",
+        "alpine3.19": "\nRUN true",
```

Comment:
```suggestion
        "alpine3.19": "",
```

Remove unnecessary `RUN true`? I expect these changes were for newline consistency. Since a newline is later appended, I expect this can be empty.